L80-csharp-load-balancing.md

```diff
@@ -0,0 +1,296 @@
+Title
```

Comment:
Looks like you need to set the title here.

Note that this is really an entire client channel design, not just about load balancing.  I suggest setting the title to something like "L80: C# .NET Client Channel Design".

Please also rename the file appropriately.

Reply:
I've updated the title to mention that connectivity and wait for ready are also here.